lib/mbta.js

```diff
@@ -2,10 +2,34 @@
 
 // Code here.
 
-const stopsBetweenStations = (startLine, startStation, endLine, endStation) => {
+let redLine = ['South Station', 'Park Street', 'Kendall', 'Central', 'Harvard', 'Porter', 'Davis', 'Alewife'];
+let greenLine = ['Government Center', 'Park Street', 'Boylston', 'Arlington', 'Copley', 'Hynes', 'Kenmore'];
+let orangeLine = ['North Station', 'Haymarket', 'Park Street', 'State', 'Downtown Crossing', 'Chinatown', 'Back Bay', 'Forest Hills'];
 
+const stopsBetweenStations = (startLine, startStation, endLine, endStation) => {
+  let stops = null;
+  if (startLine === endLine)
+    {
+        stops = startLine.indexOf(startStation) - startLine.indexOf(endStation);
+          stops = Math.abs(stops);
+        return stops;
+    }
+  else
+    {
+        let startLineStops = 0;
+        let endLineStops = 0;
+        startLineStops = (startLine.indexOf(startStation) - startLine.indexOf('Park Street'));
+          startLineStops = Math.abs(startLineStops);
+        endLineStops = (endLine.indexOf('Park Street') - endLine.indexOf(endStation));
+          endLineStops = Math.abs(endLineStops);
+        stops = startLineStops + endLineStops;
+        return stops;
```

Comment:
This is really well done. As a challenge I'd like to see you rework this into a simpler function. Notice that lines 13-15, 21-23 and 23-25 are all very similar? How would we achieve the same outcome with less code? What if someone needs to take 3 lines?